packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -753,9 +754,12 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
   private _onRenderContainer(props: IComboBoxProps): JSX.Element {
     let {
       onRenderList = this._onRenderList,
-      calloutProps
+      calloutProps,
+      calloutFixedWidth
     } = props;
 
+    let setWidth = calloutFixedWidth ? this._comboBoxWrapper.clientWidth - 2 : 'auto';
```

Comment:
@jspurlin, can we do this kind of stuff with styling instead of JavaScript? I'd like for us to avoid introducing props that are specific to styling for components that have already been converted to merge styles/glamor like combo box.

Reply:
Definitely, this should be updated to use mergeStyles instead of just setting the style here